app/client/components/cancel/stages/genericSaveAttempt.tsx

```diff
@@ -59,35 +60,71 @@ class FeedbackForm extends React.Component<
         <CaseUpdateAsyncLoader
           loadingMessage="Storing your feedback..."
           fetch={getPatchUpdateCaseFunc(this.state.feedback, this.props.caseId)}
-          render={renderFeedbackThankYou}
+          render={this.getFeedbackThankYouRenderer(this.props.reason)}
         />
       );
     }
     return (
-      <div css={{ textAlign: "right" }}>
+      <div>
+        <p>
+          {this.props.reason.alternateFeedbackIntro ||
+            "Alternatively provide feedback in the box below"}
+        </p>
         <textarea
           rows={5}
           maxLength={this.props.characterLimit}
-          css={{ width: "100%", fontSize: "inherit", fontFamily: "inherit" }}
-          onChange={this.handleChange}
-        />
-        <span
           css={{
-            fontSize: "small",
-            float: "left",
-            color: palette.neutral["3"],
-            fontFamily: sans
+            width: "100%",
+            fontSize: "inherit",
+            fontFamily: "inherit",
+            border: "1px black solid"
           }}
-        >
-          You have {this.props.characterLimit - this.state.feedback.length}{" "}
-          characters remaining
-        </span>
-        <Button
-          onClick={() => this.setState({ hasHitSubmit: true })}
-          text="Submit Feedback"
-          textColor={palette.white}
-          color={palette.neutral["4"]}
+          onChange={this.handleChange}
         />
+        <div css={{ textAlign: "right" }}>
+          <div
+            css={{
+              fontSize: "small",
+              color: palette.neutral["3"],
+              fontFamily: sans,
+              paddingBottom: "10px"
+            }}
+          >
+            You have {this.props.characterLimit - this.state.feedback.length}{" "}
+            characters remaining
+          </div>
+          <Button
+            onClick={() => this.setState({ hasHitSubmit: true })}
+            text="Submit Feedback"
+            textColor={palette.white}
+            color={palette.neutral["2"]}
+            disabled={this.state.feedback.length === 0}
+          />
+        </div>
+      </div>
+    );
+  }
+
+  private getFeedbackThankYouRenderer(
+    reason: CancellationReason
+  ): () => ReactNode {
+    return () => (
+      <div
+        css={{
+          marginLeft: "15px",
```

Comment:
Picky: this can be shorthand: `margin: "30px 0 0 15px"`

Reply:
i would rather it was explicit for readability